aspnetcore/includes/spoof.md

```diff
@@ -0,0 +1,8 @@
+  > [!WARNING]
+  > API that relies on the [Host header](https://developer.mozilla.org/docs/Web/HTTP/Headers/Host), such as <xref:Microsoft.AspNetCore.Http.HttpRequest.Host%2A?displayProperty=nameWithType> and <xref:Microsoft.AspNetCore.Builder.RoutingEndpointConventionBuilderExtensions.RequireHost%2A>, are subject to potential spoofing by clients.
+>
+> To prevent host and port spoofing, use one of the following approaches:
+>
+> * Assign a value to <xref:Microsoft.AspNetCore.Http.HttpRequest.Host?displayProperty=nameWithType> in [middleware](xref:fundamentals/middleware/write) before <xref:Microsoft.AspNetCore.Builder.EndpointRoutingApplicationBuilderExtensions.UseRouting%2A> is called.
+> * Employ [Host filtering](xref:fundamentals/servers/kestrel/host-filtering)
+> * Use <xref:Microsoft.AspNetCore.Http.HttpContext.Connection%2A?displayProperty=nameWithType> (<xref:Microsoft.AspNetCore.Http.ConnectionInfo.LocalPort?displayProperty=nameWithType>) where the ports are checked.
```

Comment:
Replacing the host doesn't make much sense.
```suggestion
> * Use <xref:Microsoft.AspNetCore.Http.HttpContext.Connection%2A?displayProperty=nameWithType> (<xref:Microsoft.AspNetCore.Http.ConnectionInfo.LocalPort?displayProperty=nameWithType>) where the ports are checked.
> * Employ [Host filtering](xref:fundamentals/servers/kestrel/host-filtering)
```

Reply:
The idea came from the following, but was worded by Halter as, "... workaround ***might be*** to ..." (emphasis added) ...

https://github.com/dotnet/aspnetcore/issues/46057#issuecomment-1380988289

I'll strike it from my Blazor NOTE after this merges.

Reply:
Oh, that's referring to something like UseForwardedHeaders to replace a internal hostname and port with the public value. That's legit.